features/developer_portal/admin/account/personal_details.feature

```diff
@@ -12,25 +12,28 @@ Feature: Dev Portal Buyer Personal Details
     And the following application:
       | Buyer | Name   |
       | John  | My App |
-    And the buyer logs in to the provider
+    And the buyer logs in
 
   @javascript
   Scenario: Buyer doesn't use current password
-    Given the buyer wants to edit their personal details
+    Given they go to the personal details page
+    And the current page is the personal details page
```

Comment:
if I see a step `they go to the personal details page`, then I assume it already checks `the current page is the personal details page`. So `the current page is the personal details page` is somehow redundant here. Just again, a thought, no request for a change.

Reply:
Now, if the step was `I click on the personal details link`, the the second step would make total sense.